recipes/hictk/conan_profiles/clang

```diff
@@ -0,0 +1,9 @@
+[settings]
+arch=x86_64
+build_type=Release
+compiler=clang
+compiler.cppstd=17
```

Comment:
If you need to use clang compiler it has to be in the build requirements I assume